BritneyS/comic-web-detail
# Grommet setup

## What does this PR do :question:
This PR uses the setup outlined [here](https://github.com/grommet/grommet-starter-new-app) to start the initial set up of the grommet UI framework for React. There are some differences: my React setup uses functional components instead of class based components, and I used the `useState` hook to manage state.


## How to test it :microscope:
1. Check out this branch.
2. Run `npm install && npm start`.
3. Open the browser to http://localhost:3000/ if the page doesn't open automatically.
4. Click the notification icon on the top right corner of the page. The sidebar section should appear.
5. Click the notification icon again. The sidebar section should disappear.
6. Resize the width of the window to its smallest size, then click the notification icon again. The sidebar section should appear over the entire screen.
7. Click the `X` button in the top right corner of the screen. The sidebar should disappear again.


## Any background info you would like to include :white_check_mark:
N/A


## Screenshots (if applicable) :camera:
![comic-web-detail-pr-grommet-setup-1](https://user-images.githubusercontent.com/8409475/60734658-a48c3c80-9f1e-11e9-8930-658a0e2539e4.gif)
![comic-web-detail-pr-grommet-setup-2](https://user-images.githubusercontent.com/8409475/60734664-a8b85a00-9f1e-11e9-9506-0a4a62a3a311.gif)
